feat: implement St-Schema integration

kidongcho opened the pull request:
- Implemented st-schema integration
 - Added skeleton template
 - polishing when conditions

kidongcho commented:
WIP : Test code for st-schema

erodewald commented:
@kidongcho Great to wake up to a PR already! The tests are failing right now, it looks like it's purely due to the linter failing code styles. 

If you're using VS Code, you can install [xo](https://marketplace.visualstudio.com/items?itemName=samverschueren.linter-xo) extension for VS Code and run the formatter. 

Before you push again, you can run `npm run lint` to test formatting, or to run everything comprehensively, run `npm test`. 

Force-push your change to update this PR and I'll test it out!

kidongcho committed:
feat: implement STtSchema integration
 - Implemented st-schema integration
 - Added st-schema skeleton template
 - polishing when conditions

kidongcho commented:
@erodewald I've been pushed with unit test after linter ```npm run lint``` and ```npm test```. Please review the commit.
